e2e-tests/e2e-fixtures/gef/constants.js

```diff
@@ -0,0 +1,37 @@
+const {
+  PORTAL_USER_ROLES: USER_ROLES,
+  PORTAL_USER_SIGN_IN_TOKENS: SIGN_IN_TOKENS,
+  DEAL_TYPE,
+  SUBMISSION_TYPE: DEAL_SUBMISSION_TYPE,
+  DEAL_STATUS,
+  FACILITY_TYPE,
+} = require('../constants.fixture');
+
+const DEAL_COMMENT_TYPE_PORTAL = {
```

Comment:
Not for this ticket, but rather once you've been on the project for a bit longer, we might want to start commonising constants like this when we come across them (it's used in `TFM-API`, so we could look to pull the `tfm-api` occurance out to `dtfs-common`, and then use the `dtfs-common` import in `tfm-api` and also in the `constants.fixture` file, then importing it into this file from `constants.fixture` -- so it all has one source of truth for this constant)

Again, not for now!